### ISSUE TITLE
Question about ICU bed usage during COVID spike

### ISSUE THREAD

**Issue by pjsample** [CareSet/COVID_Hospital_PUF#13]
I'm not sure if this is the right forum for this discussion, but I'm curious about my analysis of this data and how much sense it makes.

In short, I calculated an ICU bed usage ratio using the suggested formula (staffed_adult_icu_bed_occupancy_7_day_avg/ total_staffed_adult_icu_beds_7_day_avg) at the state level after correcting for missing values, etc.

I paired that data with weekly new COVID cases at the state level as reported by the NYTimes.

I then plotted the two metrics together and, surprisingly (?), there is essentially no correlation between ICU bed usage and new COVID cases. One might try argue that the increase in new cases is simply the result of more testing but, as we all know, deaths are dramatically higher now, as well.

So my questions are, is my analysis wrong in some way? Is the data wrong? Do hospitals try to limit the percentage of ICU beds that are used to ~80-90% so that they can handle non-COVID emergencies (just speculating here)?  

Red line = new covid cases per week
Blue line = ICU bed use ratio

<img width="1466" alt="image" src="https://user-images.githubusercontent.com/13342278/101970718-f4c20080-3be0-11eb-8620-73e1c9c1fe1a.png">

**Comment by ftrotter** [comment]
Ok, here is what I know....
First, because of data redactions and time-period shifts it will be difficult to get the ICU capacity data that is released at the facility level (i.e. this dataset) to match the state-level ICU capacity data... They are going to be  related but never simply equal. 

Having said that, you might find it easier to compare state-level ICU capacity data, with state-level case data. 

Beyond that, I recommend that you consider looking at a the ICU capacity time shifted from the new case data. Once someone is admitted with symptoms, they do not immediately get placed into an ICU, there is a lag, I have heard of success with a lag of between 7 days and 14 days... 

Let me know if these are helpful.

**Comment by daveluo** [comment]
In addition to these great suggestions by @ftrotter, I would suggest in addition to overall ICU capacity and usage which you've looked at, to also examine "How full is the adult ICU of confirmed and suspected COVID patients?" (from the example metrics section) and "What % of current ICU patients are COVID patients?" (`staffed_icu_adult_patients_confirmed_and_suspected_covid_7_day_avg` / `staffed_adult_icu_bed_occupancy_7_day_avg`).

These are illustrative ratios IMO because ICUs typically run at 80-85% utilization pre-COVID so it's the *displacement* of this typical volume and type of patients needing ICU-level care by a significant and growing portion of COVID patients that's also important to understand. These are people with non-COVID respiratory failure, heart attacks, strokes, sepsis ([ref](https://www.sccm.org/Communications/Critical-Care-Statistics#:~:text=The%20five%20primary%20ICU%20admission,severe%20sepsis%20without%20mechanical%20ventilation.)). 

In other words, if a 10-bed ICU typically manages 8 patients with these other severe health issues, what happens to those patients now that 4-5 of those ICU beds are occupied by COVID patients? It's not like patients stopped having severe heart attacks or strokes.

Also note that these reported staffed bed capacity numbers (the denominators) are inclusive of all staffed surge/expansion/overflow beds so it will go up as hospitals respond to influxes of new patients. That may also partly explain why the overall % usage of beds doesn't look to change much - because hospitals will increase capacity to treat every patient they can (and often move mountains to do so, i.e. adding more staff, converting surgical theaters to ICUs, stretching healthcare worker:patient ratios).

**Comment by pjsample** [comment]
Thanks for the thoughtful answers.

@ftrotter 
I used this data set to aggregate all ICU information and came up with state-level numbers via aggregation. And yes, there should be a lab between new cases and ICU capacity but I think that enough time has passed since we hit the exponential phase for it to be reflected. In fact, you can plot deaths, which sadly is also much higher than months earlier, instead of new cases and get the same result. (BTW, thanks for curating this data!)

@daveluo 

> In addition to these great suggestions by @ftrotter, I would suggest in addition to overall ICU capacity and usage which you've looked at, to also examine "How full is the adult ICU of confirmed and suspected COVID patients?" (from the example metrics section) and "What % of current ICU patients are COVID patients?" (staffed_icu_adult_patients_confirmed_and_suspected_covid_7_day_avg / staffed_adult_icu_bed_occupancy_7_day_avg).

Great suggestions -- I'll give that metric a go. 

The displacement of people who would otherwise be in the ICU is really concerning. Surely these are contributing to the 'excess deaths' we're seeing and I imagine we'll see a more dramatic slope in these values during this period once all of the numbers are in. So really, it seems like we may consider hospitals reporting 80 - 90% occupancy as effectively maxed out. This would make some of the numbers reported by the NYTimes (https://www.nytimes.com/interactive/2020/us/covid-hospitals-near-you.html) and others look 'better' than they actually are. Speculating here, obviously.


> Also note that these reported staffed bed capacity numbers (the denominators) are inclusive of all staffed surge/expansion/overflow beds so it will go up as hospitals respond to influxes of new patients. That may also partly explain why the overall % usage of beds doesn't look to change much - because hospitals will increase capacity to treat every patient they can (and often move mountains to do so, i.e. adding more staff, converting surgical theaters to ICUs, stretching healthcare worker:patient ratios).

I use the maximum value of the reported staffed bed occupancy at each hospital, thinking that it's probably closer to the actual max.
